src/steam/SteamClient.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using Godot;
+
+/// <summary>
+///   Concrete implementation of the Steam integration handling. Only compiled for the Steam version of the game.
+/// </summary>
+public class SteamClient : ISteamClient
+{
+    private bool initStarted;
+
+    public bool IsLoaded { get; private set; }
+    public string LoadError { get; private set; }
+    public string ExtraErrorInfo { get; private set; }
+
+    public bool? IsOnline { get; private set; }
+    public ulong? SteamId { get; private set; }
+    public bool? IsOwned { get; private set; }
+
+    public string DisplayName => Steam.GetPersonaName();
+
+    public void Init()
+    {
+        if (initStarted)
+            throw new InvalidOperationException("Can't initialize steam more than once");
+
+        initStarted = true;
+        GD.Print("Starting Steam load");
+
+        var stats = Steam.SteamInit(true);
+
+        if (!stats.Contains("status") || stats["status"] as int? != 1)
+        {
+            GD.Print("Steam load failed with code:", stats["status"]);
+            GD.PrintErr("Verbal status: ", stats["verbal"]);
+            SetError(TranslationServer.Translate("STEAM_CLIENT_INIT_FAILED"), stats["verbal"] as string);
+            return;
+        }
+
+        GD.Print("Steam load finished");
+        IsLoaded = true;
+
+        RefreshCurrentUserInfo();
+
+        if (IsOwned == true)
```

Comment:
Can it happen that somebody runs the executable via Steam without owning the game? Why is this check here?

Reply:
I mean, if you get the Steam game files, run them on your computer, and have Steam open but you don't own the game, I think that's the case where this triggers.